package.json

```diff
@@ -58,7 +58,8 @@
     "clean:website-example": "yarn workspace @wepublish/website-example clean",
     "build:website-example": "yarn workspace @wepublish/website-example build",
     "prettier": "prettier --write '{packages,services}/**/*.{js,ts,tsx,json}'",
-    "prettier-check": "prettier --check '{packages,services}/**/*.{js,ts,tsx,json}'"
+    "prettier-check": "prettier --check '{packages,services}/**/*.{js,ts,tsx,json}'",
+    "test:e2e:chromium": "node -r dotenv/config node_modules/.bin/testcafe chromium ./tests/e2e.ts"
```

Comment:
Is the idea to add a script for all popular browsers, or should we make this configurable via an environment variable?